src/main/resources/dbchangelogs/changelog-4.34.0-create-person-basedata-table.xml

```diff
@@ -0,0 +1,36 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.10.xsd">
+
+  <changeSet author="weigel" id="create_person_basedata_table">
+
+    <createTable tableName="person_basedata">
+      <column name="person_id" type="INT(10)">
+        <constraints primaryKey="true"/>
+      </column>
+      <column name="personnel_number" type="VARCHAR(255)">
+        <constraints nullable="true" />
+      </column>
+      <column name="additional_information" type="VARCHAR(255)">
```

Comment:
is varchar 255 enough for additional informations?

Reply:
I think there is [no real advantage](https://dba.stackexchange.com/a/227105) of `varchar(255)` at all. @aendi what do you think?

Reply:
We will stay with 255